File: Language_Coder/src/conditional/Code632.java
package conditional;

import java.util.Scanner;

public class Code632 {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		int num1, num2, num3, min;
		num1 = scan.nextInt();
		num2 = scan.nextInt();
		num3 = scan.nextInt();

		min = num1 < num2 ? num1 : num2;
		min = min < num3 ? min : num3;

		System.out.println(min);
	}

}

File: Language_Coder_Cpp/Code632/Code632.cpp
// Code632.cpp

#include <iostream>
using namespace std;

int main()
{
	int num1, num2, num3;

	cin >> num1 >> num2 >> num3;

	int min = num1 < num2 ? num1 : num2;
	min = min < num3 ? min : num3;

	cout << min;
}